content/source/docs/index.html.markdown

```diff
@@ -1,14 +1,95 @@
 ---
-layout: "docs"
+layout: "docs-frontpage"
 page_title: "Documentation"
-sidebar_current: "docs-home"
+sidebar_current: "docs-frontpage"
 description: |-
-  Welcome to the Terraform documentation! This documentation is more of a reference guide for all available features and options of Terraform. If you're just getting started with Terraform, please start with the introduction and getting started guide instead.
+  A brief map of the documentation for Terraform CLI, Terraform Enterprise, the
+  Terraform GitHub Actions, and the rest of the Terraform ecosystem.
 ---
 
 # Terraform Documentation
 
-Welcome to the Terraform documentation! This documentation is more of a reference
-guide for all available features and options of Terraform. If you're just getting
-started with Terraform, please start with the
-[introduction and getting started guide](/intro/index.html) instead.
+Welcome to the Terraform documentation!
+
+There are a lot of things a user of Terraform might need to know about. To help you manage this information, we've divided the Terraform docs into several sections.
+
+You can navigate the docs via this index page, or you can jump from section to section using the "Other Docs" area of the navigation sidebar, available in most areas of this site.
+
+<div class="container-fluid"><div class="row">
+
+<div class="col-md-6 col-sm-12">
+
+### [Terraform Glossary](/docs/glossary.html)
```

Comment:
I can see a rationale for having this up top, but I think it won't be most people's first stop, and would suggest shifting it down/over.